lib/rucio/common/config.py

```diff
@@ -39,7 +40,7 @@ def convert_to_any_type(value: str) -> Union[bool, int, float, str]:
     for conv in (int, float):
         try:
             return conv(value)
-        except:
+        except ValueError:
```

Comment:
Changes like this one (where you're not adding logging to a bare except, but you're specifying the exception for a generic `except`) should go in a separate commit for clarity